pkg/cmd/repo/license/list/list.go

```diff
@@ -26,10 +27,15 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 	}
 
 	cmd := &cobra.Command{
-		Use:     "list",
-		Short:   "List available repository license templates",
+		Use:   "list",
+		Short: "List available repository license templates",
+		Long: heredoc.Doc(`
+			List available repository license templates.
+			
+			Only the most commonly used licenses templates are returned. For even more licenses, visit <https://choosealicense.com/appendix>
+		`),
```

Comment:
Aside from removing the _"template"_ language, it seemed like the short and long descriptions were a little confusing if we came back and say _"Well, only the most common used ones ..."_

```suggestion
		Use:   "list",
		Short: "List common repository licenses",
		Long: heredoc.Doc(`
			List common repository licenses.
			
			For even more licenses, visit <https://choosealicense.com/appendix>
		`),
```